package.json

```diff
@@ -23,8 +23,10 @@
     "format": "yarn lint --fix"
   },
   "dependencies": {
+    "ajv": "^7.1.0",
     "avsc": ">= 5.4.13 < 6",
-    "mappersmith": ">= 2.30.1 < 3"
+    "mappersmith": ">= 2.30.1 < 3",
+    "protobufjs": "^6.10.1"
```

Comment:
@Nevon BTW, me and @dskatz22 discussed the fact that all these dependencies will be installed even if the user uses just a specific schema type.

Maybe you should consider moving the specific implementations to separate plugin packages e.g. `schema-registry-json`, `schema-registry-protobuf`, and that way the user can decide which ones to install

Reply:
Yeah I had the same thought, but I didn't want to make this PR huge and suddenly involve having to publish multiple packages.

I imagine in a future version we'll extract the different serdes implementations and have the user register them when they instantiate the schema registry client, but for now I would say we should just include them for simplicity.